README.md

```diff
@@ -37,7 +37,7 @@ Loaded!
 
 We can then ask it to solve 2 + 3 (and find the correct answer 5):
 
-```
+```text
```

Comment:
This PR makes it so that `README.md` is used in `lib.rs` as Rust doc (see below) to check the code. Rust assumes code blocks to be "rust" by default, so we need to say "text" here.